app/upload/directives/audio-version.component.ts

```diff
@@ -0,0 +1,96 @@
+import {Component, Input, OnInit, OnDestroy} from 'angular2/core';
+import {Subscription} from 'rxjs';
+import {UploadService} from '../services/upload.service';
+import {UploadFileSelect} from './upload-file-select.directive';
+import {AudioFileComponent} from './audio-file.component';
+import {HalDoc} from '../../shared/cms/haldoc';
+import {AudioModel} from '../models/audio.model';
+
+// TODO: why can't atom find this? jspm bundles it just fine
+import {Dragula, DragulaService} from 'ng2-dragula/ng2-dragula';
```

Comment:
atom typescript only searches in `node_modules` not `jspm_packages`, so the trick is in the gulp task to copy the jspm package to `node_modules`.

Then, to make atom typescript happy, you have to specify the file for the ts defs inside that module, I don't know why, I've seen it with a couple of modules before - it's weird